B4B Human Wellbeing Prediction/mlops/pipeline.py

```diff
@@ -142,10 +141,7 @@ def run(self):
         passed_threshold = custom_scoring_threshold(model, X_test, y_test)
 
         if passed_threshold:
-            explainer = shap.KernelExplainer(model.predict_proba, shap.kmeans(X_train, 15))
-            with open('../api/explainer.pkl', 'wb') as file:
-                cloudpickle.dump(explainer, file)
-
+            X_train.to_csv("../api/dataset.csv", index=False)
```

Comment:
How does to_csv create the explainer?

From what im seeing your removing shap and creating a dataset.csv? Is this intended?